app/services/category_transactions/creator.rb

```diff
@@ -0,0 +1,42 @@
+class CategoryTransactions::Creator
+  Params = Struct.new(:amount, :user_id, :category_id)
+
+  def initialize(params)
+    @params = Params.new(params[:amount], params[:user_id], params[:category_id])
+  end
```

Comment:
@marderer see comments about params in the previous PR

Reply:
@AleksSenkou If you about [this comment(https://github.com/marderer/personal-finance/pull/37#discussion_r176401112), here It will work.